docs/concepts/select.md

```diff
@@ -162,22 +133,8 @@ These options can also be provided through the `@SelectorOptions` decorator at a
 
 #### `injectContainerState`
 
-- `true` will cause all selectors defined within a state class to receive the container class' state model as their first parameter. As a result every selector would be re-evaluated after any change to that state.  
-  **NOTE:** _This is not ideal, therefore this setting default will be changing to `false` in NGXS v4._
+- `true` will cause all selectors defined within a state class to receive the container class' state model as their first parameter. As a result every selector would be re-evaluated after any change to that state.
 - `false` will prevent the injection of the container state model as the first parameter of a selector method (defined within a state class) that joins to other selectors for its parameters.
```

Comment:
We could possibly state as the first point for this option that this property is only useful for migrating codebases from NGXS v3.x and that it is not recommended to keep this set as `true`.

We should also mention that the default is now `false`.
Under the details for `true` we should mention that this should only be used during migrations.